tasks/gitlab_helpers.py

```diff
@@ -81,3 +82,45 @@ def create_gitlab_annotations_report(ci_job_id: str, ci_job_name: str):
             },
         ]
     }
+
+
+def print_gitlab_object(
+    get_object: callable, ctx, id, repo='DataDog/datadog-agent', jq: str | None = None, jq_colors=True
+):
+    """
+    Print one or more Gitlab objects in JSON and potentially query them with jq
+    """
+    repo = get_gitlab_repo(repo)
+    ids = [i for i in id.split(",") if i]
+    for id in ids:
+        obj = get_object(repo, id)
+
+        if jq:
+            jq_flags = "-C" if jq_colors else ""
+            ctx.run(f"echo '{obj.to_json()}' | jq {jq_flags} '{jq}'")
+        else:
+            obj.pprint()
+
+
+@task
+def print_pipeline(ctx, id, repo='DataDog/datadog-agent', jq: str | None = None, jq_colors=True):
+    """
+    Print one or more Gitlab pipelines in JSON and potentially query them with jq
+    """
+
+    def get_pipeline(repo, id):
+        return repo.pipelines.get(id)
+
+    print_gitlab_object(get_pipeline, ctx, id, repo, jq, jq_colors)
+
+
+@task
+def print_job(ctx, id, repo='DataDog/datadog-agent', jq: str | None = None, jq_colors=True):
```

Comment:
What does `jq` means here? Are these the `jq` parameters?

Reply:
`jq` is the JSON query passed to the `jq` executable (example: `.name` to get the name of the resulting object)